src/core/lv_global.h

```diff
@@ -31,14 +31,25 @@ extern "C" {
 #include "../stdlib/builtin/lv_tlsf.h"
 
 #if LV_USE_FONT_COMPRESSED
-#include "../font/lv_font_fmt_txt.h"
+#include "../font/lv_font_fmt_txt_private.h"
 #endif
 
 #include "../tick/lv_tick.h"
 #include "../layouts/lv_layout.h"
 
 #include "../misc/lv_types.h"
 
+#include "../misc/lv_timer_private.h"
```

Comment:
having all of the private headers included into a public header defeats the purpose of making private headers.